server/apps/publish/formatters/newsml_g2_formatter_test.py

```diff
@@ -356,95 +356,88 @@ def setUp(self):
-            self.assertEquals(xml.find(
-                '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
-                '{http://iptc.org/std/nar/2006-10-01/}rightsInfo/{http://iptc.org/std/nar/2006-10-01/}usageTerms').text,
-                'terms')
-            self.assertEquals(xml.find(
-                '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
-                '{http://iptc.org/std/nar/2006-10-01/}itemMeta/{http://iptc.org/std/nar/2006-10-01/}provider/' +
-                '{http://iptc.org/std/nar/2006-10-01/}name').text,
-                'sourcefabric.org')
-            self.assertEquals(xml.find(
-                '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
-                '{http://iptc.org/std/nar/2006-10-01/}contentMeta/{http://iptc.org/std/nar/2006-10-01/}headline').text,
-                'This is a test headline')
-            self.assertEquals(xml.find(
-                '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
-                '{http://iptc.org/std/nar/2006-10-01/}contentSet/{http://iptc.org/std/nar/2006-10-01/}inlineXML/' +
-                '{http://iptc.org/std/nar/2006-10-01/}nitf/{http://iptc.org/std/nar/2006-10-01/}body/' +
-                '{http://iptc.org/std/nar/2006-10-01/}body.content').text, 'The story body')
+        seq, doc = self.formatter.format(self.article, {'name': 'Test Subscriber'})[0]
+        xml = etree.fromstring(doc)
+        self.assertEquals(xml.find(
+            '{http://iptc.org/std/nar/2006-10-01/}header/{http://iptc.org/std/nar/2006-10-01/}sender').text,
+            'sourcefabric.org')
+        self.assertEquals(xml.find(
+            '{http://iptc.org/std/nar/2006-10-01/}header/{http://iptc.org/std/nar/2006-10-01/}origin').text, 'AAP')
+        self.assertEquals(xml.find(
+            '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
+            '{http://iptc.org/std/nar/2006-10-01/}rightsInfo/{http://iptc.org/std/nar/2006-10-01/}usageTerms').text,
+            'terms')
+        self.assertEquals(xml.find(
+            '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
+            '{http://iptc.org/std/nar/2006-10-01/}itemMeta/{http://iptc.org/std/nar/2006-10-01/}provider/' +
+            '{http://iptc.org/std/nar/2006-10-01/}name').text,
+            'sourcefabric.org')
+        self.assertEquals(xml.find(
+            '{http://iptc.org/std/nar/2006-10-01/}itemSet/{http://iptc.org/std/nar/2006-10-01/}newsItem/' +
+            '{http://iptc.org/std/nar/2006-10-01/}contentMeta/{http://iptc.org/std/nar/2006-10-01/}headline').text,
+            'This is a test headline')
+        self.assertEquals(xml.find(
```

Comment:
`assertEquals()` deprecation notice.

(will stop mentioning these in order to not shadow the essence of the changes)